web/docs/migrate-from-0-13-to-0-14.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Migration from 0.13.X to 0.14.X
+---
+
+:::note Are you on 0.11.X or earlier?
+
+This guide only covers the migration from **0.13.X to 0.14.X**. If you are migrating from 0.11.X or earlier, please read the [migration guide from 0.11.X to 0.12.X](./migrate-from-0-11-to-0-12.md) first.
+
+:::
+
+## What's new in 0.14.0?
+
+### Better auth user API
+
+We introduced a much simpler API for accessing user auth fields like `username`, `email` or `isEmailVerified` on the `user` object. You don't need to helper functions every time you want to access the user's `username` or extra steps to get proper typing.
+
+## How to migrate?
+
+### Migrate how you access user auth fields
+
+We had to make a couple of breaking changes to reach the new simpler API. 
+
+Follow the steps below to migrate:
+
+1. **Replace `getUsername` use** with `user.identities.username.id`
```

Comment:
use -> usage?

Reply:
Btw why is this so? Don't we still have `getUsername` helper function?

Reply:
Because the `AuthUser` they receive _now_ is different from the `AuthUser` they received before.

Before it was `getUsername(user: AuthUser)` and now it's `getUsername(user: UserWithAuthAndIdentities)`, so the helper no longer works on the same type of user.

Reply:
I've updated the text to mention this 👍